packages/accounts-password/package.js

```diff
@@ -5,11 +5,12 @@ Package.describe({
   // 2.2.x in the future. The version was also bumped to 2.0.0 temporarily
   // during the Meteor 1.5.1 release process, so versions 2.0.0-beta.2
   // through -beta.5 and -rc.0 have already been published.
-  version: "3.0.3",
+  version: "4.0.0",
 });
 
 Npm.depends({
   bcrypt: "5.0.1",
+  argon2: "0.41.1",

```

Comment:
should we keep using bcrypt just because the migration to Argon2 occurs during login?

Reply:
Yes, this is needed because there is no way to migrate existing users since we don't know their passwords